resources/views/news/index.blade.php

```diff
@@ -0,0 +1,30 @@
+@extends('layouts.app')
+
+@section('content')
+    <div>
+        <div>
+            <h1>News</h1>
+            @auth
+                <a href="{{ route('news.create') }}">Create new news</a>
+            @endauth
+            @if ($news->count())
+                @foreach ($news as $singleNews)
+                    <div>
+                        {{ $singleNews->title }}
+                        {!! \Illuminate\Mail\Markdown::parse($singleNews->content) !!}
+                        {{ $singleNews->author }}
+
+                        <a href="{{ route('news.edit', $singleNews) }}">Edit</a>
+                        <form action="{{ route('news.destroy', $singleNews) }}" method="post">
+                            @csrf
+                            @method('DELETE')
+                            <button type="submit"  onclick="return confirm('Delete News? This operation is irreversible.')">Delete</button>
```

Comment:
So do you send `post` or `delete` form?

Reply:
Ok, got it. Regarding to https://laravel.com/docs/9.x/blade#method-field, there is no question.